layersvt/screenshot.cpp

```diff
@@ -1029,6 +1140,7 @@ VKAPI_ATTR VkResult VKAPI_CALL QueuePresentKHR(VkQueue queue, const VkPresentInf
             fileName = base + ".ppm";
 #else
             fileName = to_string(frameNumber) + ".ppm";
+            printf("Screen Capture file is: %s \n", fileName.c_str());
```

Comment:
I will address this one later with a different change: as printing the directory instead of filenames. Lets keep it as is for now